lib/model/query/users.js

```diff
@@ -39,8 +39,10 @@ const updatePassword = (user, cleartext) => ({ run, bcrypt }) =>
     .then((hash) => run(sql`update users set password=${hash} where "actorId"=${user.actor.id}`));
 updatePassword.audit = (user) => (log) => log('user.update', user.actor, { data: { password: true } });
 
-const invalidatePassword = (user) => ({ run }) =>
-  run(sql`update users set password=null where "actorId"=${user.actor.id}`);
+const invalidatePassword = (user) => ({ Sessions, run }) => Promise.all([
+  run(sql`update users set password=null where "actorId"=${user.actor.id}`),
+  Sessions.terminateByActorId(user.actorId)
```

Comment:
I thought about adding this to the resource rather than this query method, but I think it makes sense to group these two queries together. The resource is the only place that calls this query method.